dashboard/final-example/app/dashboard/layout.tsx

```diff
@@ -6,7 +6,7 @@ export default function Layout({ children }: { children: React.ReactNode }) {
       <div className="w-full flex-none md:w-64">
         <SideNav />
       </div>
-      <div className="flex-grow p-6 md:overflow-y-auto md:p-12">{children}</div>
+      <div className="grow p-6 md:overflow-y-auto md:p-12">{children}</div>
```

Comment:
Nice catch on this one 👏🏼 

Could you update the code snippet in the layouts chapter? 🙏🏼

Reply:
I have a draft PR with it! Making one other edit before it'll be ready for review! I'll post it in the channel soon